docs/dev/framework/response-context.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Response Context"
+description: "The Response Context is a concept special to Contao."
+alias:
+  - /framework/response-context/
+---
+
+{{< version "4.12" >}}
+
+This section describes the "Response Context" concept. A concept that is special
+to Contao and thus deserves its own section in the documentation.
+
+## Motivation
+
+Content Management Systems do not share all the same properties as static site generators, or your
+good old Symfony based website.
+In a traditional Symfony application, you as the developer control 100% of the output of
+a particular HTML website. Speaking in Symfony terms, your controller 
+controls the full `Response` instance, from its HTTP headers to it's content, everything from
+the opening `<html>` to the closing `</html>`, all the scripts, the stylesheets, the page title and
+the description.
+
+In Contao, however, most of the content is dynamically created. That's why it's a Content Management
+System after all. Let's take e.g. a news reader front end module as an example. It displays the 
+detail of a certain news entry and thus also would like to adjust the `<title>` tag in the `<head>`
+section of the HTML document. The controller, however, does not know if that news reader front end module
```

Comment:
```suggestion
section of the HTML document. The controller however does not know if that news reader front end module
```

Reply:
Are you sure? I was taught to separate `however` with commas.

Reply:
Yeah, you are right. Even when it is used for emphasis, commans are usually put around it, though apparently there is no clear rule. So leave it in ;)